Gemfile

```diff
@@ -1,5 +1,7 @@
 source "https://rubygems.org"
 
+gem "refinerycms-authentication-devise", '~> 1.0.4'
```

Comment:
This is fine, by the way, if the library works without it and we're just using it to test when it's present.